README.markdown

```diff
@@ -12,6 +12,10 @@ Note: `nvm` does not support [Fish] either (see [#303](https://github.com/creati
  - [nvm](https://github.com/derekstavis/plugin-nvm) plugin for [Oh My Fish](https://github.com/oh-my-fish/oh-my-fish), which makes nvm and its completions available in fish shell
  - [bass](https://github.com/edc/bass) allows to use utilities written for Bash in fish shell
 
+Note: We still have some problem with FreeBSD, because there is no pre-built binary from oficial for FreeBSD, and build from source may need [patches](https://www.freshports.org/www/node/files/patch-deps_v8_src_base_platform_platform-posix.cc), see the issue ticket:
```

Comment:
sorry i didn't notice this sooner; `s/problem/problems/`, `s/oficial/official`, `s/build/building`. Also, this is kind of a run-on sentence :-)